test/integration/scheduler_perf/README.md

```diff
@@ -36,6 +36,17 @@ make test-integration WHAT=./test/integration/scheduler_perf ETCD_LOGLEVEL=warn
 ```
 
 The benchmark suite runs all the tests specified under config/performance-config.yaml.
+By default, it runs all workloads that have the "performance" label. In the configuration,
+labels can be added to a test case and/or individual workloads. Each workload also has
+all labels of its test case. The `perf-scheduling-label-filter` command line flag can
+be used to select workloads. It works like GitHub label filtering: the flag accepts
+a comma-separated list of label names. Each label may have a + or - as prefix. Labels with
++ or no prefix must be set for a workload for it to be run. - means that the label must not
```

Comment:
nits:

```suggestion
a comma-separated list of label names. Each label may have a `+` or `-` as prefix. Labels with
`+` or no prefix must be set for a workload for it to be run. `-` means that the label must not
```

Reply:
> I'm wondering if this PR's goal is to solely filter the tests running on 5000 nodes. 

No, the intention is to go beyond that. I also want the ability to define workloads that normally don't run in the Prow job. In the past that was done by commenting them out, but then running them manually must undo that first. With labels, such workloads can be defined without the "performance" label.

A future use case might be to run other, smaller workloads as part of pull-kubernetes-integration. I'm working on enabling "go test -race" for that (currently blocked by actual data races in other code), which then will have the advantage of increasing test coverage also for the production code.

I've updated the README.md as suggested.

Reply:
Makes sense to me. Thanks @pohly .